tests/jballerina-integration-test/src/test/resources/testng.xml

```diff
@@ -146,125 +147,6 @@
         </classes>
     </test>
 
-    <test name="ballerina-transaction-tests" parallel="false">
```

Comment:
Why we remove these test cases?

Reply:
That was by mistake. Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/23549/commits/46166159f4e6e31e94bf3c66d2b85f8443f4adc5